src/Sumtypes.hs

```diff
@@ -345,6 +339,15 @@ copyCase typeEnv env concreteStructTy@(StructTy _ typeVariables) (theCase, isFir
   let (name, tys, correctedTagName) = namesFromCase theCase concreteStructTy
   in unlines
      [ "    " ++ (if isFirstCase then "" else "else ") ++ "if(pRef->_tag == " ++ correctedTagName ++ ") {"
-     , joinLines (map (memberCopy typeEnv env) (zip (map (\anon -> name ++ "." ++ anon) anonMemberNames) tys))
+     , joinLines $ memberCopy typeEnv env <$> unionMembers name tys
      , "    }"
      ]
+
+anonMemberName :: String -> String -> String
```

Comment:
What issue does this resolve? (mainly curious)

Reply:
Anonymous unions aren't allowed in C99. Sumtype cases are now accessed as `.u.memberX`.

Reply:
OK, I see. Thanks!